### PR TITLE
Add CCG confidence method spec.

### PR DESCRIPTION
**Pull request by brianorwhatever**
Adding confidence method spec being incubated at CCG.

### CODE REVIEW

**Comment by brianorwhatever** [comment]
@awoie @decentralgabe FYI added you 2 as maintainers here

File: vc/confidence-method/.htaccess [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# VC Render Method Contexts
```
Comment (davidlehn) [inline vc/confidence-method/.htaccess:1]: Fix copy & paste issue.

Comment (TallTed) [reply vc/confidence-method/.htaccess:1]:
```suggestion
# VC Confidence Method Contexts
```

**Comment by brianorwhatever** [comment]
sorry about the copy paste issue. should be fixed up now. thanks @davidlehn and @TallTed

**Pull request merged**